website/static/js/pages/sharing-page.js

```diff
@@ -2,7 +2,7 @@
 var $ = require('jquery');
 
 var ContribManager = require('js/contribManager');
-var ContribAdder = require('js/contribAdder');
```

Comment:
I like this refactor, moving `ContribAdder` down into `ContribManager`, to be joined by a `ContribRemover`.